app/models/manageiq/providers/inventory/persister/builder/cloud_manager.rb

```diff
@@ -123,11 +123,11 @@ def orchestration_stack_ancestry_save_block
 
           model_class = stacks_inventory_collection.model_class
 
-          stacks_parents_indexed = model_class.select(%i(id ancestry))
+          stacks_parents_indexed = model_class.select(%i[id ancestry miq_group_id])
                                               .where(:id => stacks_parents.values).find_each.index_by(&:id)
 
           ActiveRecord::Base.transaction do
-            model_class.select(%i(id ancestry))
+            model_class.select(%i[id ancestry miq_group_id])
```

Comment:
validate: this says that when you see `miq_group_id` then lets just have it local, don't try and ancestry-ize it

Reply:
This says that since we have a callback on these that runs code in the ownership mixin and relies on these objects having group ids, when we create these test objects, we'd better be sure that that group id is present in the list of fields. Otherwise these fail with an error about missing attributes. It's the last set of messages in your gitter: 
```
ActiveModel::MissingAttributeError: missing attribute: miq_group_id
from /Users/marjorie/.rvm/gems/ruby-2.6.7/gems/activemodel-6.0.3.7/lib/active_model/attribute.rb:221:in `value'
[6] pry(#<ManageIQ::Providers::Amazon::CloudManager::OrchestrationStack>)> method(:valid?).source_location
=> ["/Users/marjorie/.rvm/gems/ruby-2.6.7/gems/activerecord-6.0.3.7/lib/active_record/validations.rb", 66]
[7] pry(#<ManageIQ::Providers::Amazon::CloudManager::OrchestrationStack>)> self
=> #<ManageIQ::Providers::Amazon::CloudManager::OrchestrationStack:0x00007fb30119bf70 id: 31000000000920, ancestry: "31000000000919">
```

Reply:
ok. Think we should change ancestry itself to not use validity on the whole object to determine validity of this column.

it is strange though. we always checked validity on this column.

Reply:
If you'd like to make that change in Ancestry, I'd be more than happy to take out this change here just as soon as we're using the relevant Ancestry version, capisce?

Reply:
wait - if you change the host to the service, is this change still needed. I really do not want this change
Still can't tell if this is a hack to get our tests running or if it is something else.

Reply:
I'm going to have to look into this one, unclear why it had to be changed

Reply:
The ancestry pr is https://github.com/stefankroes/ancestry/pull/542

Reply:
@agrare this is a very nuanced error - one that I'm glad has been raised. But to be honest, I'd like to either chat or voice this one as it requires quite a bit of understanding of the value of the relationship_type among all the callbacks - need to take into account that the value changes depending upon which object you are accessing. easy for it to slip by while in the debugger.